.github/workflows/ci.yaml

```diff
@@ -299,9 +299,6 @@ jobs:
       - name: Setup Node
         uses: ./.github/actions/setup-node
 
-      - name: Check Go version
```

Comment:
I think we still want this to ensure people don't mess up upgrades right?

Reply:
I think my editor did something wonky when it dc/rc, good catch